### ISSUE TITLE
Introduce fixture aliases in gherkin style

### ISSUE THREAD

**Issue by raniejade** [JetBrains/spek#799]
The following fixture aliases will be introduced:
- `Scenario.beforeScenario` -> `Scenario.beforeGroup`
- `Scenario.afterScenario` -> `Scenario.afterGroup`
- `Scenario.beforeEachStep` -> `Scenario.beforeEachTest`
- `Scenario.afterEachStep` -> `Scenario.afterEachTest`
- `Feature.beforeEachScenario` -> `Feature.beforeEachGroup`
- `Feature.afterEachScenario` -> `Feature.afterEachGroup`
- `Feature.beforeFeature` -> `Feature.beforeGroup`
- `Feature.afterFeature` -> `Feature.afterGroup`

The aliased fixtures will be deprecated in the context of the `gherkin` style. This can be done by:

```kotlin
class Feature(val delegate: GroupBody): LifecycleAware by delegate {
   ...

   @Deprecated(..., replaceWith = ReplaceWith("beforeEachScenario"))
   override fun beforeEachGroup(fixture: Fixture) { delegate.beforeGroup(fixture) }
}
```

**Comment by kamilmasyhur** [comment]
I see that you develop 2.1.0, already 28%
are you going to develop this? @raniejade

**Comment by raniejade** [comment]
I have not worked on the `2.1.0` milestone for quite a while now, it mostly contains task for supporting Kotlin Native. This will be included in the next release (`2.0.9`)
